payments/tests/factories.py

```diff
@@ -68,15 +65,19 @@ class AdditionalProductFactory(AbstractBaseProductFactory):
     period = factory.LazyFunction(lambda: PeriodType.SEASON)
     tax_percentage = factory.LazyFunction(lambda: DEFAULT_TAX_PERCENTAGE)
 
-    # Because of the FIXED_SERVICE restrictions (only allowed to have 24% VAT)
-    # the actual assignment of a random Tax value is done once the service has
-    # been assigned to the model.
     @factory.post_generation
-    def service_taxes(self, created, extracted, **kwargs):
+    def price_value_correction(self, created, extracted, **kwargs):
```

Comment:
should this be just `price_value`? should it also be aware of the `extracted`?

Reply:
The problem with naming it `price_value` is that it ignores the value assigned to the actual `price_value` by the factory/generator since it's a `post_generation` hook, so when the object is being created, `self.price_value == None` which causes the DB insertion to fail.

Reply:
UPD: we went with a `LazyAttribute` instead 🤗